src/screens/HyperSwitch/Analytics/PaymentsAnalytics/PaymentAnalyticsEntity.res

```diff
@@ -448,6 +488,56 @@ let getStatData = (
       statType: "Volume",
       showDelta: false,
     }
+  | RetriesCount => {
+      title: "Smart Retries made",
+      tooltipText: "Total number of retries that were attempted after a failed payment attempt",
+      deltaTooltipComponent: AnalyticsUtils.singlestatDeltaTooltipFormat(
+        singleStatData.retries_count->Belt.Int.toFloat,
+        deltaTimestampData.currentSr,
+      ),
+      value: singleStatData.retries_count->Belt.Int.toFloat,
+      delta: {
+        singleStatData.retries_count->Belt.Int.toFloat
+      },
+      data: constructData("retries_count", timeSeriesData),
+      statType: "Volume",
+      showDelta: false,
+    }
+  | RetriesAmountProcessed => {
+      title: `Smart retries savings`,
+      tooltipText: "Total number of retries that were attempted after a failed payment attempt",
```

Comment:
tooltip description is same for both `RetriesCount`and `RetriesAmountProcessed` is same